packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form.feature

```diff
@@ -0,0 +1,107 @@
+      | GBP      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | AUD      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | CAD      | Canada                                                       |
+      | EUR      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | JPY      | Japan                                                        |
+
+  Scenario Outline: Payment Method text is dynamic depending on the currency and country selected
+    Given the bank account payment method is selected
+    And the selected currency is <currency>
+    And the selected bank country is one of <bankCountry>
+    Then <paymentMethodText> is shown
+    And the withdrawal fee is shown
+
+    Examples: 
+      | currency | bankCountry                                                  | paymentMethodText |
+      | USD      | United States                                                | EFT Withdrawal    |
+      | USD      | Canada, Spain, Ireland, United Kingdom, Japan                | FX Wire           |
+      | GBP      | United Kingdom                                               | EFT Withdrawal    |
+      | GBP      | United States, Canada, Spain, Ireland, Japan                 | FX Wire           |
+      | AUD      | United States, Canada, Spain, Ireland, United Kingdom, Japan | FX Wire           |
+      | CAD      | Canada                                                       | EFT Withdrawal    |
+      | EUR      | Spain, Ireland,  United Kingdom                              | EFT Withdrawal    |
+      | EUR      | United States, Canada, Japan                                 | FX Wire           |
+      | JPY      | Japan                                                        | EFT Withdrawal    |
+    # Not sure what the actual specs for when it's shown are yet
+
+  Scenario Outline: PayPal option is dynamically shown
+    Given <currency> is set on the payout partner
+    And the currency <currencySupported> supported by PayPal
+    Then the paypal option <mayBe> shown as a payment method
+
+    Examples: 
+      | currency | currencySupported | mayBe |
+      | USD      | is                | is    |
+      | GBP      | is                | is    |
+      | AUD      | is                | is    |
+      | CAD      | is                | is    |
+      | EUR      | is                | is    |
+      | JPY      | is                | is    |
+
+  Scenario: Error state is shown if the participant tries to submit the form before all the fields have been filled
+    Given the participant has selected a payment method
+    And they have not filled out all of the required fields
+    When they submit the form
+    Then there will be a red border with a message underneath each of the inputs that are missing from the form
+
+  @minutia
+  Scenario: A loading error banner appears when the form fails to load
```

Comment:
THis spec is just malformed, scenario but with examples but only 1 example?

Reply:
yeah, i don't think we should have specs using text that is editable as well